doc/changes/changes_3.0.0.md

```diff
@@ -1,33 +1,23 @@
-# Virtual Schema for document data in files 3.0.0, released 2021-??-??
+# Virtual Schema for document data in files 3.0.0, released 2021-11-22
 
-Code name:
+Code name: Improved Parquet Loading Performance
+
+## Summary
+
+In this release we improved the performance for loading parquet files. The improvements will give you a speed-up when you load a few (<200) files.
```

Comment:
Maybe also change this to "fewer than available CPUs"?

Reply:
200 is hard coded here. If it's fewer than 200 the files are distributed explicitly otherwise by hash binning.

Reply:
Ahh yes, this for the other improvement.